Library/Formula/binkd.rb

```diff
@@ -0,0 +1,21 @@
+class Binkd < Formula
+  desc "TCP/IP FTN Mailer"
+  homepage "http://binkd.grumbler.org/"
+  url "ftp://happy.kiev.ua/pub/fidosoft/mailer/binkd/binkd-1.0.4.tar.gz"
+  sha256 "917e45c379bbd1a140d1fe43179a591f1b2ec4004b236d6e0c4680be8f1a0dc0"
+
+  def install
+    cp Pathname.glob("mkfls/unix/*").select(&:file?), "."
```

Comment:
Use `Dir["mkfls/unix/*"]` instead of `Pathname.glob`. Why is this step necessary?